### ISSUE TITLE
'Clear all' not working when footer is disabled

### ISSUE THREAD

**Issue by rvdsteege** [p0deje/Maccy#865]
### Before Submitting Your Bug Report

- [X] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

2.0.0.beta.14

### macOS Version

15.0 beta (24A5331b)

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonKeyCode\\":35,\\"carbonModifiers\\":2048}";
    "KeyboardShortcuts_popup" = "{\\"carbonModifiers\\":2304,\\"carbonKeyCode\\":9}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSStatusItem Visible Item-0" = 0;
    "NSStatusItem Visible Item-1" = 0;
    "NSWindow Frame SUStatusFrame" = "1080 982 400 134 0 0 2560 1415 ";
    "NSWindow Frame SUUpdateAlert" = "970 784 620 398 0 0 2560 1415 ";
    "NSWindow Frame com.sindresorhus.Preferences.FrameAutosaveName" = "625 398 542 435 0 0 1792 1095 ";
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "925 723 510 487 0 0 2560 1415 ";
    SUAutomaticallyUpdate = 1;
    SUEnableAutomaticChecks = 1;
    SUHasLaunchedBefore = 1;
    SULastCheckTime = "2024-09-09 09:06:47 +0000";
    SUUpdateGroupIdentifier = 1244922666;
    SUUpdateRelaunchingMarker = 1;
    WebKitDefaultFontSize = 13;
    WebKitJavaScriptEnabled = 0;
    WebKitPluginsEnabled = 0;
    WebKitStandardFont = "-apple-system-font";
    WebKitUserStyleSheetEnabledPreferenceKey = 1;
    WebKitUserStyleSheetLocationPreferenceKey = "/Applications/Maccy.app/Contents/Frameworks/Sparkle.framework/Resources/ReleaseNotesColorStyle.css";
    avoidTakingFocus = 1;
    enabledPasteboardTypes =     (
        "public.html",
        "public.rtf",
        "public.utf8-plain-text"
    );
    highlightMatch = color;
    history = {length = 42, bytes = 0x62706c69 73743030 a0080000 00000000 ... 00000000 00000009 };
    historySize = 150;
    hotKey = "command+option+v";
    ignoreRegexp =     (
    );
    ignoredPasteboardTypes =     (
        "com.agilebits.onepassword",
        "de.petermaurer.TransientPasteboardType",
        "Pasteboard generator type",
        "com.typeit4me.clipping",
        "net.antelle.keeweb"
    );
    imageMaxHeight = 16;
    maxMenuItemLength = 60;
    maxMenuItems = 20;
    migrations =     {
        "2024-07-01-version-2" = 1;
    };
    pasteByDefault = 1;
    pinTo = top;
    popupPosition = cursor;
    previewDelay = 250;
    removeFormattingByDefault = 0;
    searchMode = exact;
    showFooter = 1;
    showInStatusBar = 0;
    showRecentCopyInMenuBar = 0;
    showSearch = 1;
    showSpecialSymbols = 1;
    showTitle = 0;
    sortBy = lastCopiedAt;
    suppressClearAlert = 1;
    windowPosition = "[0.578515625,0.6095890410958904]";
    windowSize = "[450,503]";
}
```

### Description

Clearing all items with the ⌥⇧⌘⌫ keyboard shortcut seems not to work when `showFooter = 0`.

### Steps to Reproduce

1. Disable footer
2. Try clearing all items with the ⌥⇧⌘⌫ keyboard shortcut

**Comment by p0deje** [comment]
Fixed in 2.0.1

**Comment by rvdsteege** [comment]
Thank you, @p0deje!
